support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -72,9 +72,7 @@ object CheckoutValidationRules {
       if (switches.directDebit.isOn) Valid else Invalid("Invalid Payment Method")
     case Left(_: SepaPaymentFields) =>
       if (switches.sepa.isOn) Valid else Invalid("Invalid Payment Method")
-    case Left(_: StripeSourcePaymentFields) =>
-      if (switches.stripe.isOn) Valid else Invalid("Invalid Payment Method")
-    case Left(s: StripePaymentMethodPaymentFields) =>
+    case Left(s: StripePaymentFields) =>
```

Comment:
I got rid of `StripeSourcePaymentFields` as it was for the old non-SCA checkouts which don't exist.  I renamed StripePaymentMethodPaymentFields to StripePaymentFields as it's the only one now.